### DISCUSSION TITLE
How to make robots.txt prevent all scraping

### DISCUSSION THREAD

**Discussion by TimyIsCool** [searxng/searxng#4065] [Q&A] [answered]
So I would like to host my own searxng instance but would like it so google and whatnot doesn't pick it up within search results
I did some digging around and saw something about it but nothing from what I can see in the settings to say what to allow and disallow in docker
Thanks for any help!

**Comment by xlionjuan** [comment, marked as answer]
I wanna know about this

My first mind is use Docker volume strategy to "override" `robots.txt`
but after some search, I found this:

https://github.com/searxng/searxng/blob/ab1e895cc03df639e8a7bee28f23fb06bfa3125a/searx/webapp.py#L1219-L1228

It is dynamic responded by Python, it is impossible without modify the code, which is unrealistic.
